source/standards/operating-systems.html.md.erb

```diff
@@ -1,21 +1,40 @@
 ---
 title: Operating systems for virtual machines
-last_reviewed_on: 2019-06-18
+last_reviewed_on: 2019-09-19
 review_in: 3 months
 ---
 
 # <%= current_page.data.title %>
 
-You should use [Ubuntu Long Term Support (LTS)](https://www.ubuntu.com/download/server) as an operating system (OS) for your virtual machine (VM).
+### Amazon Web Services Virtual Machines
```

Comment:
Can we make this an h2? It's not accessibility best practice to go straight from an h1 to h3.